checkov/common/sca/commons.py

```diff
@@ -50,6 +50,19 @@ def get_package_lines(package: dict[str, Any]) -> list[int] | None:
     return package.get("linesNumbers", package.get("lines"))
 
 
+def get_record_file_line_range(package: dict[str, Any], file_line_range: list[int] | None) -> list[int]:
+    """
+    Currently, there are 2 way for getting file_line_range for the sca-report:
+    1. by the arg 'file_line_range' which is arrived from the runner - this is specific for entire file (e.g: image referencer)
+    2. by a dedicated attribute in a package-object - (e.g: DT-cli V2)
+    The purpose of this function is making sure there are no conflicts between those resources, and return a valid rage
+    """
+    package_line_range = get_package_lines(package)
+    if package_line_range and file_line_range:
+        raise Exception('Both \'package_line_range\' and \'file_line_range\' are not None. Conflict.')
```

Comment:
do we really want to raise an exception or just log a message?

Reply:
I prefer raising exepction. it isn't valid behavior. otherwise we can't have bad logic without knowing about it

Reply:
fixed